contrib/toolchains/toolchain-linux-ggp-release.cmake

```diff
@@ -0,0 +1,16 @@
+set(CMAKE_BUILD_TYPE
+    "Release"
+    CACHE STRING "build type")
```

Comment:
Just comparing with toolchain-linux-default-release.cmake without being sure what its effect is: why is FORCE missing here?

Reply:
Having a set(... CACHE) means it's only set when it's not overwritten by the user. Having the FORCE option means, it's set even if the user tries to (accidentally) overwrite it.

In this case, I wanted the user to be able to use the very same toolchain file, even if they build a debug build. Since this toolchain file does not contain any build-type specific compiler flags,
I thought that's the right way to go here.

Reply:
Thanks for the explanation!